#!/usr/bin/env python
PACKAGE = "robotx_vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_area", 			int_t,    0, "An Integer parameter", 300,	0,10000)
gen.add("low_threshold", 		int_t,    0, "An Integer parameter", 0,	0,10000)
gen.add("high_threshold",		int_t,    0, "An Integer parameter", 1500,	0,10000)

exit(gen.generate(PACKAGE, "robotx_vision", "shape"))
